src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -2041,11 +2205,12 @@ public void removeOnMixpanelTweaksUpdatedListener(OnMixpanelTweaksUpdatedListene
 
     ////////////////////////////////////////////////////
     protected void flushNoDecideCheck() {
```

Comment:
i've never used this guy. what does it do exactly?

Reply:
flushes events/people queues without hitting decide